domains/athanasopoulos.json

```diff
@@ -1,11 +1,11 @@
 {
   "description": "A personal website, for me.",
-  "repo": "https://github.com/athanasso/athanasso.github.io",
   "owner": {
     "username": "athanasso",
     "email": "manos.athanasopoulos@yahoo.gr"
   },
   "record": {
-    "CNAME": "athanasso.github.io"
+    "TXT": "vc-domain-verify=athanasopoulos.is-a.dev,56d11a7196dfe6d19e95",
+    "CNAME": "next-portfolio-eight-kohl.vercel.app/"
```

Comment:
Please read the docs on how to use Vercel.
Also you don't have to open PRs every 1 minute